Sources/PrivoSDK/model/ParentChildPair.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+enum RoleIdentifier: String {
+    case parentStandard = "STANDARD_PARENT_1"
+    case childDefault = "DEFAULT_CHILD"
+}
+
+struct ParentChildPair: Encodable {
+    let roleIdentifier: String
+    let email: String
+    let minorRegistrations: [MinorRegistration]
+    
+    struct MinorRegistration: Encodable {
+        let userName: String
+        let firstName: String
+        let lastName: String?
+        let birthDateYYYYMMDD: String
+        let sendParentEmail: Bool
+        let roleIdentifier: String
+        let sendCongratulationsEmail: Bool
+        let attributes: [Attribute]
+        
+        init(child: Child) {
+            self.firstName = child.firstname
+            self.lastName = child.lastname
+            self.userName = child.username
+            let dateFormatter = DateFormatter()
+            dateFormatter.dateFormat = "yyyyMMdd"
```

Comment:
dateFormat = "yyyyMMdd" it's cold birthDatePrecision 
please add it to struct as a separate field. See example here - https://consent-dev.privo.com/#/demo

Reply:
implemented